src/components/navigation-header/components/dropdown-menu/dropdown-menu.module.css

```diff
@@ -1,146 +1,152 @@
 .root {
-  width: fit-content;
-  position: relative;
+	width: fit-content;
+	position: relative;
 }
 
 .activatorWrapper {
-  padding-bottom: 16px;
-  padding-top: 16px;
+	padding-bottom: 16px;
+	padding-top: 16px;
 }
 
 .activator {
-  /* Composition */
-  composes: g-focus-ring-from-box-shadow-dark from global;
-
-  /* CSS properties */
-  align-items: center;
-  background-color: transparent;
-  border-radius: 5px;
-  border: 0;
-  display: flex;
-  padding-bottom: 8px;
-  padding-left: var(--header-dropdown-menu-padding-left-right);
-  padding-right: var(--header-dropdown-menu-padding-left-right);
-  padding-top: 8px;
-
-  &[aria-expanded='true'] {
-    & .activatorText {
-      color: var(--token-color-surface-primary);
-    }
-
-    & .activatorTrailingIcon {
-      color: var(--token-color-surface-primary);
-      transform: rotate(-180deg);
-    }
-  }
+	/* Composition */
+	composes: g-focus-ring-from-box-shadow-dark from global;
+
+	/* CSS properties */
+	align-items: center;
+	background-color: transparent;
+	border-radius: 5px;
+	border: 0;
+	display: flex;
+	padding-bottom: 8px;
+	padding-left: var(--header-dropdown-menu-padding-left-right);
+	padding-right: var(--header-dropdown-menu-padding-left-right);
+	padding-top: 8px;
+
+	&[aria-expanded='true'] {
+		& .activatorText {
+			color: var(--token-color-surface-primary);
+		}
+
+		& .activatorTrailingIcon {
+			color: var(--token-color-surface-primary);
+			transform: rotate(-180deg);
+		}
+	}
 }
 
 .activatorText {
-  color: var(--token-color-palette-neutral-400);
-  margin-right: 6px;
+	color: var(--token-color-palette-neutral-400);
+	margin-right: 6px;
 }
 
 .activatorLeadingIcon {
-  margin-right: 4px;
+	margin-right: 4px;
 }
 
 .activatorTrailingIcon {
-  color: var(--token-color-palette-neutral-400);
+	color: var(--token-color-palette-neutral-400);
 
-  /* Only enable animation if query is supported and value is no-preference */
-  @media (prefers-reduced-motion: no-preference) {
-    transition: transform 0.2s ease-in-out;
-  }
+	/* Only enable animation if query is supported and value is no-preference */
+	@media (prefers-reduced-motion: no-preference) {
+		transition: transform 0.2s ease-in-out;
+	}
 }
 
 .dropdownContainer {
-  background-color: white;
-  border-bottom-left-radius: 6px;
-  border-bottom-right-radius: 6px;
-  box-shadow: var(--token-elevation-higher-box-shadow);
-  min-width: 252px;
-  padding: 16px;
-  position: absolute;
+	background-color: white;
+	border-bottom-left-radius: 6px;
+	border-bottom-right-radius: 6px;
+	box-shadow: var(--token-elevation-higher-box-shadow);
+	min-width: 252px;
+	padding: 16px;
+	position: absolute;
 }
 
 .itemGroupDivider {
-  margin-bottom: 20px;
-  margin-top: 20px;
+	margin-bottom: 20px;
+	margin-top: 20px;
 }
 
 .itemGroup {
-  list-style: none;
-  margin: 0;
-  padding: 0;
+	list-style: none;
+	margin: 0;
+	padding: 0;
+
+	&.twoColumns {
+		column-count: 2;
+		width: max-content;
+		gap: 24px;
+	}
```

Comment:
This is the meaningful change in this file - the styles for our two-column layout option. The rest is formatting.